PyRDF/backend/Utils.py

```diff
@@ -20,4 +20,5 @@ def declare_headers(cls, includes):
 
         """
         for header in includes:
-            ROOT.gInterpreter.Declare("#include \"{}\"\n".format(header))
\ No newline at end of file
+            if not ROOT.gInterpreter.Declare("#include \"{}\"\n".format(header)):
+                raise Exception("There was an error in including \"{}\". Please fix the error and retry !".format(header))
```

Comment:
I would suppress the "Please fix the error and retry" part of the message. More information will be printed by cling.